pkg/cedar/intake/translation/biz.go

```diff
@@ -135,9 +137,20 @@ func (bc *TranslatableBusinessCase) CreateIntakeModel() (*wire.IntakeInput, erro
 		}
 		lc.Cost = pStr(cost)
 
-		obj.LifecycleCostLines = append(obj.LifecycleCostLines, lc)
+		if line.Solution == models.LifecycleCostSolutionPREFERRED {
+			preferredSolution.LifecycleCostLines = append(preferredSolution.LifecycleCostLines, lc)
+		} else if line.Solution == models.LifecycleCostSolutionA {
+			alternativeASolution.LifecycleCostLines = append(alternativeASolution.LifecycleCostLines, lc)
+		} else if line.Solution == models.LifecycleCostSolutionB {
+			alternativeBSolution.LifecycleCostLines = append(alternativeBSolution.LifecycleCostLines, lc)
+		}
 	}
 
+	// Append all solution objects to business solutions list
```

Comment:
These lines aren't needed; the solutions get appended to `obj.BusinessSolutions` right after each one is initialized (lines 74, 96, 115).

Reply:
Yeah thats a mistake, good catch. You think its more readable/cleaner to append them all at the end after we finish appending the LifecycleCostLines or just append them when they are created (lines 74, 96, and 115).

I do think it's a little weird to append the objects to the business solutions and then edit them afterwards. Thoughts?

Reply:
I went ahead and made the change of removing the business solution appends on lines 74, 96, 115. If anyone has any major issues with it im happy to revert to the old pattern. I figured this translation will probably change again once we modify our db to hold LifecycleCosts in the BusinessSolutions as well

Reply:
LGTM now; I slightly favor doing it all in one block, it's more clear to me.

Incidentally, appending then editing doesn't make a semantic difference here, because `preferredSolution`, `alternativeASolution`, and `alternativeBSolution` are pointers. It would be a difference if they were non-pointers - editing after appending wouldn't work in that case. So editing then appending (like it is now) is more clear in general.